src_kt/javatests/arcs/util/Base64PerformanceTest.kt

```diff
@@ -29,6 +30,7 @@ import kotlin.system.measureNanoTime
 /**
  * Performance comparison tests between Arcs' implementation of Base64-encoding/decoding and Java's.
  */
+@Ignore // Remove to run manually.
```

Comment:
Maybe add a reason why you turned it off? e.g. "performance too variable on Travis"